ajax/libs/zuix/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "zuix",
+  "filename": "zuix.min.js",
+  "version": "1.0.3",
+  "description": "A JavaScript library for creating component-based websites and applications.",
+  "keywords": [
+    "webapp",
+    "bundler",
+    "pwa",
+    "components",
+    "mvc",
+    "dom",
+    "frontend",
+    "modular-web"
+  ],
+  "homepage": "https://zuixjs.org",
+  "author": {
+    "name": "Generoso Martello",
+    "email": "gene@glabs.it",
+    "url": "https://glabs.it"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zuixjs/zuix.git"
+  },
+  "npmName": "zuix-dist",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "zuix?(.min).+(js|map)"
```

Comment:
Is this a regex? Should the `?` appear after the `(.min)`?

Reply:
These are globs iirc, not regex.

Reply:
![image](https://user-images.githubusercontent.com/12371363/68624411-aec2d680-04ce-11ea-8498-b18c2dd2379a.png)
http://www.globtester.com/#p=eJyrKs2ssNfQy83M09TT1sgqrslNLNAEAE%2BABxQ%3D&r=eJyzScksU0jOSSwutlXKTSxJzlCyqyrNrNDLKrbRB0rZ2eCQz83MI0pNbmIBpiKofElqcYlNUpEdNgUIcQApKTg4&

Reply:
This is awesome, thank you for showing it to me @MattIPv4